lib/printer.js

```diff
@@ -1300,9 +1300,14 @@ function genericPrintNoParens(path, options, print) {
             parts.push(": ");
         }
 
+        var params = path.map(print, "params");
+        if (n.rest) {
+            params.push(concat(["...", path.call(print, "rest")]));
```

Comment:
The `concat([...])` is not necessary here; `params.push("...", path.call(print, "rest"))` should work.